### ISSUE TITLE
"Star" a module

### ISSUE THREAD

**Issue by chillu** [silverstripe/addons.silverstripe.org#12]
Similar to the github approach. Acts as a "favourite modules" list, from which we can build up, e.g. customized RSS feeds, new version alerts, etc.

Dev note: Since we don't allow login, a lightweight approach for this would be browser localstorage.

**Comment by camfindlay** [comment]
What about SSO using GitHub account to keep track of favs?

**Comment by chillu** [comment]
@camfindlay Yeah, I think SSO with Github could be useful in many aspects of addons. Although I'd keep the actual "star" storage in SilverStripe. Github has a similar concept, but not all addons will be on Github.
